io.github.FailurePoint.RandomNumberFive.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "io.github.FailurePoint.RandomNumberFive",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "random-number-five",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "random-number-five",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/FailurePoint/RandomNumberFive.git"
```

Comment:
a commit or tag is required.

Reply:
How do I find that? I am still pretty new to the GitHub system.

Reply:
Get the commit from the version used; git will contain it, and GitHub will expose it in the UI.